packages/@react-spectrum/color/docs/ColorSlider.mdx

```diff
@@ -0,0 +1,228 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/color';
+import colorTypes from 'docs:@react-types/color/src/index.d.ts';
+import statelyDocs from 'docs:@react-stately/color';
+import {HeaderInfo, PropTable, TypeLink} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/color/package.json';
+
+```jsx import
+import {ColorSlider} from '@react-spectrum/color';
+import {Flex} from '@react-spectrum/layout';
+```
+
+---
+category: Color
+keywords: [color slider]
+---
+
+# ColorSlider
+
+<p>{docs.exports.ColorSlider.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ColorSlider']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/color-slider/'}
+  ]} />
+
+## Example
+
+```tsx example
+<ColorSlider defaultValue="#7f0000" channel="red" />
+```
+
+## Value
+
+A ColorSlider requires either a uncontrolled default value or a controlled value, provided using the `defaultValue` or `value` props respectively.
+The value provided to either of these props should be a string or <TypeLink links={colorTypes.links} type={colorTypes.exports.Color} />  object that matches the provided ColorSlider's `channel` prop. For example, a
+ColorSlider representing a color channel ("red", "green", "blue") would have a hex color code (e.g. "#7f0000") provided. For a full list of supported channels,
```

Comment:
I don't really understand what you are trying to say here. Can you reword/elaborate?

Reply:
For which bit specifically?
If for the Color object bit:
The type for `value` is "string | Color" where `Color` is defined as [this](https://github.com/adobe/react-spectrum/blob/main/packages/@react-types/color/src/index.d.ts#L47-L77). This Color object can come from using `parseColor` (which people may use for controlled ColorSliders since onChange returns a Color object).

If for the matching color channel, the string/object needs to match (e.g. if channel="red"/"green"/"blue", the string would need to be a hex color value. If the channel was "hue" then a hsl value e.g. `hsl(0, 100%, 50%)` or a hsb value e.g. `hsb(0, 100%, 50%)`)

Reply:
You could also use a rgb object `rgb(100, 10, 10)` instead of hex which is maybe where my confusion comes from?

Reply:
Ah, derp, thats right. I'll add that to line 50